app/src/main/java/com/psiphon3/psiphonlibrary/MainBase.java

```diff
@@ -522,10 +450,6 @@ public boolean onTouch(View v, MotionEvent event) {
             findViewById(R.id.sponsorWebView).setOnTouchListener(onTouchListener);
             findViewById(R.id.statisticsView).setOnTouchListener(onTouchListener);
             findViewById(R.id.settingsView).setOnTouchListener(onTouchListener);
-            findViewById(R.id.regionSelector).setOnTouchListener(onTouchListener);
-            findViewById(R.id.tunnelWholeDeviceToggle).setOnTouchListener(onTouchListener);
```

Comment:
I think we need something here to enable swiping on the Options tab.

Reply:
Yes, it is missing currently, thanks for reminding.